libc/stdio_file.c

```diff
@@ -65,7 +65,18 @@ TEST_SETUP(stdio_fopenfclose)
 
 
 TEST_TEAR_DOWN(stdio_fopenfclose)
-{ /* empty */
+{
+	/* if the filep is NULL it means that file does not exist or is closed */
```

Comment:
please write in line 48 that these variables are global to gracefully close opened files in case of test failure

(still, I don't like that these values are global, I think I'm ok with failing test possibly breaking other tests in the same if it makes code prettier)